code/modules/sanity/sanity_mob.dm

```diff
@@ -359,8 +359,12 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
 					resting_times = 1
 				for(var/stat in L)
 					var/stat_up = L[stat] * 2 * resting_times
-					to_chat(owner, SPAN_NOTICE("Your [stat] stat goes up by [stat_up]"))
-					owner.stats.changeStat(stat, stat_up)
+					if((owner.stats.getStat(stat)) >= STAT_VALUE_MAXIMUM)
+						stat_up = 0
+						to_chat(owner, SPAN_NOTICE("Your [stat] can not be raised any higher with simple trinkets..."))
```

Comment:
This sounds rather confusing since you don't have any 'different' sort of trinkets. Would be great to change to
'You feel that you can't grow anymore better for today in [stat]'